hooks/useCurrencyPrice.ts

```diff
@@ -0,0 +1,21 @@
+import { useRecoilValue } from 'recoil';
+
+import useSynthetixQueries from '@synthetixio/queries';
+import { CurrencyKey } from 'constants/currency';
+import { priceCurrencyState } from 'store/app';
+import { zeroBN } from 'utils/formatters/number';
+import { wei } from '@synthetixio/wei';
+
+const useCurrencyPrice = (currencyKey: CurrencyKey) => {
```

Comment:
do we need a query for the price? I think we have this already

Reply:
seconding. thoughts?